roborazzi-compose/src/main/java/com/github/takahirom/roborazzi/RoborazziComposePreview.kt

```diff
@@ -117,7 +124,24 @@ private fun (@Composable () -> Unit).withSize(
       heightDp > 0 -> Modifier.height(heightDp.dp)
       else -> Modifier
     }
-    Box(modifier = modifier) { this@withSize() }
+    Box(modifier = modifier) {
+      this@withSize()
+    }
   }
   return resizedPreview
+}
+
+private fun registerRoborazziComposePreviewsActivityToRobolectricIfNeeded() {
+  try {
+    val appContext: Application = ApplicationProvider.getApplicationContext()
+    Shadows.shadowOf(appContext.packageManager).addActivityIfNotPresent(
+      ComponentName(
+        appContext.packageName,
+        ComponentActivity::class.java.name,
```

Comment:
Should this be RoborazziComposePreviewsActivity?

Reply:
I have not delete the RoborazziCompoaePreviewsActivity because I’m still experimenting😅.
Anyway, I still get the same result with both unfortunately… seems the height is not exactly what it should by some pixels 🧐